### ISSUE TITLE
build: warnning issue after updated xcode9

### ISSUE THREAD

**Issue by tommyZZM** [Microsoft/ChakraCore#3779]
OS: macOS 10.12.6
Xcode: 9.0 (just updated)
Command Line Tools (macOS Sierra version 10.12) for Xcode: 9.0

```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/
usr/include/c++/v1/__undef_min_max:29:2: 
warning: : macro max is incompatible with C++.  #undefing max [-W#warnings]
```

many of these warning is happen on building

**Comment by dilijev** [comment]
@obastemur Can you take a look?

**Comment by obastemur** [comment]
@tommyZZM this is just a warning right?

**Comment by tommyZZM** [comment]
hi @obastemur 
it seems just a warnning, `ch` is runable after build

**Comment by obastemur** [comment]
@tommyZZM Thanks for sharing this with us. The problem should be fixed on release/1.7 and (soon) master branches.
